internal/pipe/publish/publish.go

```diff
@@ -37,7 +36,6 @@ type Publisher interface {
 // nolint: gochecknoglobals
 var publishers = []Publisher{
 	s3.Pipe{},
-	blob.Pipe{},
```

Comment:
merge conflict still? 

this should be here...

Reply:
fixed in 5155302